cfpb/consumerfinance.gov
# Improve layout of printed links, hide printed hero images

This PR:

- Hides hero images when printing. They're background images and thus don't print by default, resulting in a giant block of white space.
- Makes button links and jump/list links look more like standard links when printing.
- Displays the shortened version of Ask CFPB links when printing.

## Changes

- I changed the data attribute from `orig-href` to `pretty-href` to better reflect the data it's storing -- a prettier version of the href.


## How to test this PR

1. Print a [page](http://localhost:8000/complaint/) with a button link and confirm it's no longer gray with weird padding.
1. Print a [sublanding page](http://localhost:8000/consumer-tools/auto-loans/) and confirm there's no longer a giant gap holding the place of the hero background image.
1. Print an [ask page](http://localhost:8000/ask-cfpb/what-is-a-credit-card-interest-rate-what-does-apr-mean-en-44/) and confirm links to other ask pages use their shortened `askcfpb/XXXX` URLs and that the list links are condensed and lack borders.


## Screenshots

| before                 | after |
|------------------------|-------|
| **button links**<br> <br><img width="530" alt="before" src="https://user-images.githubusercontent.com/1060248/111686060-fcf09400-87fe-11eb-8fbf-4ff386e50c75.png"> | **button links**<br> <br><img width="530" alt="after" src="https://user-images.githubusercontent.com/1060248/111686070-ff52ee00-87fe-11eb-98ff-94e4778cc972.png"> |
| **jump links and ask cfpb links**<br> <br><img width="532" alt="befire" src="https://user-images.githubusercontent.com/1060248/111687265-76d54d00-8800-11eb-977b-77e77f931568.png"> | **jump links and ask cfpb links**<br> <br><img width="511" alt="after" src="https://user-images.githubusercontent.com/1060248/111687282-7a68d400-8800-11eb-93e9-f008bdb51fda.png"> |
| **hero images**<br> <br><img width="657" alt="before" src="https://user-images.githubusercontent.com/1060248/111686415-6a042980-87ff-11eb-9762-bf94f78dc841.png"> | **hero images**<br> <br><img width="657" alt="after" src="https://user-images.githubusercontent.com/1060248/111686424-6c668380-87ff-11eb-931c-ce22e6e2e87d.png"> |













## Notes and todos

-


## Checklist



  - PR titles are used to generate the change log in [releases](../../releases); good ones make that easier to scan.
  - Consider prefixing, e.g., "Mega Menu: fix layout bug", or "Docs: Update Docker installation instructions".
  - [This repo’s docs](https://cfpb.github.io/consumerfinance.gov/) (edit the files in the `/docs` folder) – for basic, close-to-the-code docs on working with this repo
  - CFGOV/platform wiki on GHE – for internal CFPB developer guidance
  - CFPB/hubcap wiki on GHE – for internal CFPB design and content guidance

### Front-end testing



#### Browser testing

Visually tested in the following supported browsers:



#### Accessibility


#### Other
